<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtmessagebox</class>
 <widget class="QWidget" name="vtmessagebox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Qwidget{
background-color:transparent;
}</string>
  </property>
  <widget class="QLabel" name="lb_dis">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
    font-size:13px;
	font-family:UTM Avo;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYes">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>25</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size:13px;
	font-weight:normal;
    background-color:transparent;
	color:#efefef;
    border:1px solid #efefef;
	padding-bottom:3px;
	font-family:UTM Avo;
}

QPushButton::hover {
    background-color:#16a085;
}

QPushButton::pressed {
    background-color:#2ecc71;
}</string>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>25</y>
     <width>75</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-size:13px;
	font-weight:normal;
    background-color:transparent;
	color:#efefef;
    border:1px solid #efefef;
	padding-bottom:3px;
	font-family:UTM Avo;
}

QPushButton::hover {
    background-color:#16a085;
}

QPushButton::pressed {
    background-color:#2ecc71;
}</string>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
